lib/logstash/codecs/identity_map_codec.rb

```diff
@@ -0,0 +1,255 @@
+# encoding: utf-8
+require "logstash/namespace"
+# require "concurrent"
+require "thread_safe"
```

Comment:
Well lest just create an issue with it and remove the comment, I don't expect a new version of concurrent-ruby to be out before we release that fix.